tests/misc/test_uefi_boot/README.rst

```diff
@@ -0,0 +1,50 @@
+UEFI Boot
+###########
+
+This test is used to verify the UEFI bootable method on x86_64 platforms.
+
+
+Requirements
+############
+
+Please install uefi-run in your system environment according to this
+reference link https://github.com/Richard-W/uefi-run.
+
+Please install OVMF in your system environment according to this
+reference link https://github.com/tianocore/tianocore.github.io/wiki/OVMF.
```

Comment:
Are there matching PRs to add these as Zephyr modules?  Having manually-installed external dependencies is sort of OK for an oddball board, but this is a CI platform.  Seems like this shouldn't be too hard.

Reply:
@andyross  for CI, I have a PR zephyrproject-rtos/docker-image#68 to add those dependencies into CI docker image, then CI could run this test, for developers, I think they should install it by themselves.

for running this test, we only need final binary uefi-run and ovmf.fd, so do we need to add them as a zephyr module and build them from source code? actually, I didn't build them from source code, I downloaded the release binary files directly.